en/docs/Learn/ConsumeAPI/ManageSubscription/AdvancedTopics/adding-an-api-subscription-workflow.md

```diff
@@ -165,23 +168,31 @@ First, enable the API subscription workflow **.**
         ...
     </WorkFlowExtensions>
     ```
+
     !!! tip
         **Note** that all workflow process services of the EI/BPS run on port 9765 because you changed its default port (9763) with an offset of 2.
 
 
         The application creation WS Workflow Executor is now engaged.
 
 
-3.  Go to the API Store Web interface and subscribe to an API.
-    It invokes the API subscription process and creates a Human Task instance that holds the execution of the BPEL until some action is performed on it.
-4.  Note the message that appears if the BPEL is invoked correctly, saying that the request is successfully submitted.
+3.  Go to the API Devportal credentials page and subscribe to an API. It will invoke the API subscription process and creates a Human Task instance that holds the execution of the BPEL until some action is performed on it. After subscribing you will see the subscription status as ON_HOLD
```

Comment:
It will trigger the API subscription process and create a Human Task instance that pauses the execution of the BPEL until an action is performed on it.